integration/js/utils/WebdriverSauceLabs.js

```diff
@@ -104,9 +104,7 @@ const getSauceLabsConfig = (capabilities) => {
     tags: [capabilities.name],
     seleniumVersion: '3.141.59',
     tunnelIdentifier: process.env.JOB_ID,
-    ...(capabilities.platform.toUpperCase() !== 'LINUX' && {
-      extendedDebugging: true
-    })
+    ...(capabilities.platform.toUpperCase() !== 'LINUX')
```

Comment:
You can remove this as well, I think the `extendedDebugging` was in place for NON linux platform hence the check.

Reply:
Last time we do this it does not solve the issue. Since this only limited to content share now can we only disable it for content share to see if it does any better?